### ISSUE TITLE
Got error "Network Error (tcp_error)" when saving video with duration more than 3 second

### ISSUE THREAD

**Issue by webfreaks** [PHP-FFMpeg/PHP-FFMpeg#290]
| Q              | A
| -------------- | ---
| Bug?           | no
| New Feature?   | no
| Version Used   | 0.7.0
| FFmpeg Version | git-2016-11-26-99ee8ee
| OS             | CentOS 7

Hi,

I got an error when saving a video with duration more than 3 seconds. I thought it's about timeout then I set FFMPEG timeout to 18000000. But still get the same error.

Is this problem fully from my server or I did something wrong?

The error says:
```
Network Error (tcp_error) 

A communication error occurred: "Operation timed out"
The Web Server may be down, too busy, or experiencing other problems preventing it from responding to requests. You may wish to try again at a later time.

For assistance, contact your network support team.
```

Thank you.

**Comment by schmanat** [comment]
It is realy a issue with this library? What did the Server Log says?

**Comment by Romain** [comment]
I completely agree with @schmanat, logs are mandatory to understand the problem.
Could you paste them here, along with your code?
Thanks
